cypress/integration/main_page_spec.js

```diff
@@ -2,23 +2,57 @@
 describe ('Main Page', () => {
 
     beforeEach(() => {
-        cy.visit('http://localhost:3000');
-      });
-
-    describe('Main Page Load', () => {
+        cy.fixture('all_movie_data.json')
+          .then((movies) => {
+            cy.intercept('GET','https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
```

Comment:
we can do something similar for error tests.... changing the status codes to 400/500 and display the respective message